### PR TITLE
Updates for v10 release

### PR DESCRIPTION
**Pull request by macmv**
Updates things to release the shell for v10. Changes include:

- Change default eval format to use isatty in all versions
- Default to v10 for fauna shell and fauna eval
- Update description and examples of eval/shell
- Update README for v10

### CODE REVIEW

**Review by breinero — CHANGES_REQUESTED** [review]
Only a minor grammatical nit, but otherwise LGTM

**Review by freels — APPROVED** [review]
@breinero feels like the mini-tutorial could be moved/replaced with docs. At the very least it deserves a refresh to showcase v10 features.

**Review by freels — CHANGES_REQUESTED** [review]
Actually there are quite a few more FaunaDB references that need to be fixed.

File: README.md [hunk 1]
```diff
@@ -163,134 +163,130 @@ my_app>
 Once you have the prompt ready, you can start issues queries against your FaunaDB instance. (Note that the results shown here might vary from the ones you see while running the examples).
```
Comment (freels) [inline README.md:163]:
```suggestion
Once you have the prompt ready, you can start issues queries against your Fauna database. (Note that the results shown here might vary from the ones you see while running the examples).
```

File: docs/dev.md [hunk 2]
```diff
@@ -4,36 +4,36 @@ This is an attempt to describe what are the project goals, and give an architect
...
+3. Help people connect to Fauna (cloud or self hosted)
 
 The user should have an easy way to handle the configuration to the various endpoints they might want to connect to.
 
 For that the shell offers a file called `.fauna-shell`, stored at their home directory, where `fauna` will store the connection parameters for the endpoints.
 
 Adding an endpoint should be as easy as typing: `$ fauna add-endpoint "https://example.org:443"`.
 
-Connecting to the FaunaDB Cloud should be as easy as typing: `$ fauna cloud-login`. The user will be prompted for their email & passwords, and their key will be saved in the `.fauna-shell` file.
+Connecting to the Fauna Cloud should be as easy as typing: `$ fauna cloud-login`. The user will be prompted for their email & passwords, and their key will be saved in the `.fauna-shell` file.
```
Comment (freels) [inline docs/dev.md:32]: minor:

```suggestion
Connecting to Fauna should be as easy as typing: `$ fauna cloud-login`. The user will be prompted for their email & passwords, and their key will be saved in the `.fauna-shell` file.
```

**Review by freels — APPROVED** [review]

**Pull request merged**